includes/definitions/discovery/datacom-dm1205e.yaml

```diff
@@ -0,0 +1,11 @@
+mib: SNMPv2-MIB
+modules:
+    os:
+     hardware: 
+             - SNMPv2-MIB::sysDescr.0
```

Comment:
https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery
Use sysDescr_regex instead, what you have done here is call snmp to the device again, when you already have sysDescr

Reply:
Just a question. How can I use the information fetched by the sysDescr in hardware field without call the snmp again? Sometimes I don't have the hardware description in another OID then I use the sysDescr.